cmd/install.go

```diff
@@ -0,0 +1,126 @@
+package cmd
```

Comment:
This should be in it's own package, like the CF/MR/GH download commands; maybe with a root command named something like `url`? Then the command would be `packwiz url install`